proposals/2024-11-16_qml_interface.md

```diff
@@ -0,0 +1,256 @@
+to extra information without requiring to change the layout or overload it when unused. Here are some examples
+
+_Hotcue edit_\
+![./contents/2024-11-16_qml_interface_overlays_hotcue.svg](./contents/2024-11-16_qml_interface_overlays_hotcue.svg)
+
+_Beatgrid edit_\
+![./contents/2024-11-16_qml_interface_overlays_beatgrid.svg](./contents/2024-11-16_qml_interface_overlays_beatgrid.svg)
+
+_Playlist details_\
+![./contents/2024-11-16_qml_interface_overlays_playlist.svg](./contents/2024-11-16_qml_interface_overlays_playlist.svg)
+
+### Search bar provide more capability of the engine leveraging the engine
+
+I recently stumbled across the search syntax on the documentation and discover that Mixxx had a fairly powerful search
+capability. Sadly, the simple text input in our current themes doesn't make that very clear and requires a less
+friendly experience using some bespoke search syntax (or as I call it, the "mixxxql"). With the new overlay approach,
+we cwe could imagine a search experience more interactive and dynamic:
+
+![./contents/2024-11-16_qml_interface_overlays_search.svg](./contents/2024-11-16_qml_interface_overlays_search.svg)
+![./contents/2024-11-16_qml_interface_overlays_search4.svg](./contents/2024-11-16_qml_interface_overlays_search4.svg)
+![./contents/2024-11-16_qml_interface_overlays_search3.svg](./contents/2024-11-16_qml_interface_overlays_search3.svg)
+![./contents/2024-11-16_qml_interface_overlays_search2.svg](./contents/2024-11-16_qml_interface_overlays_search2.svg)
+
+### Add more interactions on settings and preferences
+
+Here is a suggestion on how the audio routing could look like:
+
+The general UX behavior as currently imagined is:
+
+* Point-n-click/touch an edge, then another, or drag and drop between edge
+* Long click or context click on existing edge to undo them
+
+![./contents/2024-11-16_qml_interface_setting_audio_routing.svg](./contents/2024-11-16_qml_interface_setting_audio_routing.svg)
+
+Here is a suggestion on how controllers could be reworked
+
+![./contents/2024-11-16_qml_interface_setting_controller_overview.svg](./contents/2024-11-16_qml_interface_setting_controller_overview.svg)
+![./contents/2024-11-16_qml_interface_setting_controller_settings.svg](./contents/2024-11-16_qml_interface_setting_controller_settings.svg)
+![./contents/2024-11-16_qml_interface_setting_controller_pointnclick.svg](./contents/2024-11-16_qml_interface_setting_controller_pointnclick.svg)
+Here is an example of how this could be done for waveform, allowing changes feedback directly in  the setting, rather
+than on the UI itself, as it is currently. This would simplify massively the current waveform legacy and remove the
+need for the `WaveformFactory`
+
+![./contents/2024-11-16_qml_interface_setting_waveform.svg](./contents/2024-11-16_qml_interface_setting_waveform.svg)
+
+**Bonus**: [The whole Figma design](
+![./contents/2024-11-16_qml_interface_global.svg](./contents/2024-11-16_qml_interface_global.svg))
```

Comment:
the current link gives a 404
https://github.com/mixxxdj/proposals/blob/73645c45f02cecc1b6900c75703bdc1fbb663018/proposals/!%5B./contents/2024-11-16_qml_interface_global.svg%5D(./contents/2024-11-16_qml_interface_global.svg)

should be
https://github.com/acolombier/mixxx-proposals/blob/new-qml-interface/proposals/contents/2024-11-16_qml_interface_global.svg

I think this works (didn't want to edit your md file directly)
```suggestion
./contents/2024-11-16_qml_interface_global.svg)
```